client/post-editor/media-modal/detail/detail-fields.jsx

```diff
@@ -82,16 +86,30 @@ class EditorMediaModalDetailFields extends Component {
 		this.props.onUpdate( this.props.item.ID, this.state.modifiedItem );
 	}
 
-	setFieldValue = ( { target } ) => {
+	setFieldByName = ( name, value ) => {
 		const modifiedItem = Object.assign(
 			{ ID: this.props.item.ID },
 			get( this.state, 'modifiedItem', {} ),
-			{ [ target.name ]: target.value }
+			{ [ name ]: value }
 		);
 
 		this.setState( { modifiedItem }, this.persistChange );
 	};
 
+	setFieldValue = ( { target } ) => {
+		this.setFieldByName( target.name, target.value );
+	};
+
+	handleRatingChange = ( { currentTarget } ) => {
+		this.setFieldByName( 'rating', currentTarget.value );
+	};
+
+	handleDisplayEmbed = () => {
+		const inputValue = '1' === this.getItemValue( 'display_embed' ) ? '0' : '1';
```

Comment:
🤔 Can you expand what is doing this line? Why `inputValue` is the opposite of `display_embed`?

Reply:
Oh, it's clear after reading the lines below. Do you think we can improve this by renaming the function to make clear this is handling when user toggles/changes the sharing?

Something like `toggleDisplayEmbed` or `handleDisplayEmbedChange` could work.